scripts/lib/libvirt/fixtures/cloud-node1-uefi.xml

```diff
@@ -5,8 +5,8 @@
   <vcpu>1</vcpu>
   <os>
     <type arch='x86_64' machine='pc-0.14'>hvm</type>
-<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-ms-code.bin</loader>
-    <nvram>/var/lib/libvirt/cloud-node1_VARS.fd</nvram>
+<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-code.bin</loader>
```

Comment:
Supposedly, we could skip creating the uefi image and use a generic signed one, pointing to our grub.cfg, which is the part that its important to us (to add the crowbar key, discovery options and so on) but I havent had any luck with testing that.